### DISCUSSION TITLE
Is there a way to attach to the stdin of a process?

### DISCUSSION THREAD

**Discussion by sbdchd** [F1bonacc1/process-compose#51] [Q&A]
I think for like 90% of time I don't need stdin, but every once and a while I want to run the debugger in a Django app for instance which requires a terminal / stdin

**Comment by F1bonacc1** [comment]
Hi @sbdchd,

It is not supported right now.
Is this something that you would primarily use in TUI mode?

**Reply by sbdchd** [reply]
Yeah I think it would only be a TUI

I like to use ipdb typically

**Comment by dzmitry-lahoda** [comment]
If I could send STDIN via HTTP API, would add step to remote runs may be.

**Comment by F1bonacc1** [comment]
It should be possible in v1.85.0
https://f1bonacc1.github.io/process-compose/interactive-processes/
Your feedback is more than welcome

**Reply by arathunku** [reply]
Yay, this is amazing, thank you @F1bonacc1!

My only piece of feedback:

I think right now the only thing missing is some kind of customization for C-a (`tcell.KeyCtrlA`) or using one of the F keys for easy focus switch. In my case C-a is already occupied by tmux, but it's not a dealbreaker, one can always use send-keys

**Reply by F1bonacc1** [reply]
Thank you for the feedback!
I will add the ability to configure the modifier key combination in the next release.
